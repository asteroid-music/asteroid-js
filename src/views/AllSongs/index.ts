//Main component
export { AsteroidSongList } from './AllSongs';


