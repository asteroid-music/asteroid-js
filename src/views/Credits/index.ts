//Main component
export { AsteroidCredits } from './Credits';


