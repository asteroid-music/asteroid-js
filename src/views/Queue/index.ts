//Main component
export { AsteroidQueue } from './Queue';


