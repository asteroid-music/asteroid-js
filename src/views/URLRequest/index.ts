//Main component
export { AsteroidURLRequester } from './URLRequest';


