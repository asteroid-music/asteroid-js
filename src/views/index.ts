export { AsteroidCredits } from './Credits';
export { AsteroidQueue } from './Queue';
export { AsteroidURLRequester } from './URLRequest';
export { AsteroidSongList } from './AllSongs';


