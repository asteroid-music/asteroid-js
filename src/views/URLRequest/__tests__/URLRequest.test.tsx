import React from 'react';
import renderer from 'react-test-renderer';
import { AsteroidURLRequester as URLRequest } from 'views/URLRequest';

describe('<URLRequest />',()=>{
    //Basic rendering with minimal props
    describe('basic rendering',()=>{
        it("renders correctly",()=>{
            expect(renderer
                .create(<URLRequest/>)
                .toJSON()
            ).toMatchSnapshot();
        });
    });
});