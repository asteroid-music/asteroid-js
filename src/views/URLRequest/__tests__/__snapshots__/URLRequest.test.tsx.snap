// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<URLRequest /> basic rendering renders correctly 1`] = `
<Material UI Container>
  <Material UI Typography
    variant="body1"
  >
    Insert the URL to request here!
  </Material UI Typography>
  <Material UI TextField
    id="filled-basic"
    label="Input URL"
    onChange={[Function]}
    style={
      Object {
        "width": "100%",
      }
    }
    value=""
    variant="filled"
  />
  <button
    onClick={[Function]}
  >
    Material UI Button with props:
    Object {
  "children": " Request ",
  "onClick": [Function],
}
  </button>
</Material UI Container>
`;
